frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -1534,8 +1535,10 @@ class SegmentsView extends React.Component<Props, State> {
     }
     const groupToSegmentsMap = createGroupToSegmentsMap(segments);
     const relevantGroupIds = getGroupByIdWithSubgroups(segmentGroups, groupId);
-    const segmentIdsNested = relevantGroupIds.map((groupId) => groupToSegmentsMap[groupId]);
-    return segmentIdsNested.flat();
+    const segmentIdsNested = relevantGroupIds
+      .map((groupId) => (groupToSegmentsMap[groupId] != null ? groupToSegmentsMap[groupId] : null))
+      .filter((x) => x !== null);
+    return segmentIdsNested.filter((x) => x).flat() as Segment[];
```

Comment:
I think, one filter operation should suffice. I'd use `x != null` (not `!==`) as a predicate. This will cover null and undefined.